pkg/labels/fake_applicator.go

```diff
@@ -117,6 +118,79 @@ func (app *fakeApplicator) WatchMatches(selector labels.Selector, labelType Type
 	return ch
 }
 
+func (app *fakeApplicator) WatchDiff(labelType Type, quitCh <-chan struct{}) <-chan *LabeledChanges {
```

Comment:
Updated, all daemon set processes will now use a WatchMatchDiff function which uses the WatchMatches as a helper function to detect changes.